DependencyInjection/MonologExtension.php

```diff
@@ -668,6 +668,51 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
             ));
             break;
 
+        case 'sentry':
+            if (null !== $handler['client_id']) {
+                $clientId = $handler['client_id'];
+            } else {
+                $options = new Definition(
+                    'Sentry\\Options',
+                    array(array('dsn' => $handler['dsn']))
+                );
+
+                if (!empty($handler['environment'])) {
+                    $options->addMethodCall('setEnvironment', [$handler['environment']]);
+                }
+
+                if (!empty($handler['release'])) {
+                    $options->addMethodCall('setRelease', [$handler['release']]);
+                }
+
+                $builder = new Definition('Sentry\\ClientBuilder', array($options));
+
+                $client = new Definition('Sentry\\Client');
+                $client->setFactory(array($builder, 'getClient'));
+
+                $clientId = 'monolog.sentry.client.'.sha1($handler['dsn']);
+                $container->setDefinition($clientId, $client);
+
+                if (!$container->hasAlias('Sentry\\Client')) {
+                    $container->setAlias('Sentry\\Client', $clientId);
+                }
+            }
+
+            $hub = new Definition(
+                'Sentry\\State\\Hub',
+                array(new Reference($clientId))
+            );
+
+            // can't set the hub to the current hub, getting into a recursion otherwise...
+            //$hub->addMethodCall('setCurrent', array($hub));
+
+            $definition->setarguments(array(
```

Comment:
```suggestion
            $definition->setArguments(array(
```

Reply:
FFS, I thought I fixed all of them. Doesn't look like it, but it's now fixed :P